### PR TITLE
Update claim status unavailable alert

### PR DESCRIPTION
**Pull request by molindoz**
Closes: [Sunsets #155](https://github.com/department-of-veterans-affairs/sunsets-team/issues/155)

screen shot 2016-11-15 at 2 45 43 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@cagedcrown Sorry, the issue might not have been clear from that GH ticket. We want to change the message just for when a user doesn't have access to the service. Right now we show `ClaimsUnavailable` for both the app being unavailable and when a user doesn't have access to disability benefits. The logic for that is [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx#L25). We want to have the old message for when the app is not available and Mel's new message for when `canUseApp` is false.

**Comment by cagedcrown** [comment]
@jbalboni Ah! Sorry about that. I understand now and I'll make the change. Thanks.

File: src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx [hunk 1]
```diff
@@ -22,7 +30,8 @@ function AppContent({ authorized, available, synced, children, isDataAvailable }
     <div className="disability-benefits-content">
       {available && authorized && canUseApp && !synced && <ClaimSyncWarning/>}
       {!authorized && <div className="row"><div className="columns medium-8"><ClaimsUnauthorized/></div></div>}
-      {authorized && (!available || !canUseApp) && unavailableView}
+      {authorized && !available && unavailableView}
+      {authorized && !canUseApp && unavailableMviRecords}
```
Comment (jbalboni) [inline src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx]: We might want to check that this one is available, just to make sure we don't get duplicate messages:

`authorized && !canUseApp && available && unavailableMviRecords`

**Review by jbalboni — APPROVED** [review]
LGTM!

**Pull request merged**